migration-from-4x.md

```diff
@@ -335,7 +335,33 @@ The following configurations options have changed in the Custom Menu items in Ti
 * `addMenuItem` has a new configuration.
 * A new method `addNestedMenuItem` has been added to the options. The `addNestedMenuItem` is an explicit method for creating menu items that have a submenu with more menu items.
 * A new method `addToggleMenuItem` has been added to the options. The `addToggleMenuItem` is an explicit method for creating toggle menu items similar to the new special [toolbar button methods]({{site.baseurl}}/ui-components/typesoftoolbarbuttons/).
-* The concept of `context` has been removed from menu item configurations.
+* The concept of `context` has been removed from menu item configurations. The [`menu` setting]({{site.baseurl}}/configure/editor-appearance/#menu) provides this functionality for TinyMCE 5.
+
+To add a custom item to a menu, use the `menu` setting. All items in the menu need to be declared in order to appear.
```

Comment:
Maybe change "the menu" to "a menu"? We distinguish menus and the menubar internally, but I'm not sure others would as easily. So I think "a" might make it clearer what we're referring to than "the"?

Reply:
@blackcathikari, as in:
>To add a custom item to _the_ menu,
?